src/main/java/com/transportcompany/transportcompany/models/dtos/VehicleDTO.java

```diff
@@ -0,0 +1,10 @@
+package com.transportcompany.transportcompany.models.dtos;
+
+import lombok.Data;
+
+@Data
+public class VehicleDTO {
+    private long id;
```

Comment:
Please don't forget about formatting, we should have an empty row between the class name and the fist line of code.